src/GetModuleExtension.ps1

```diff
@@ -5,10 +5,24 @@ function Get-ModuleExtension {
         $ModuleName,
 
         [Version]
-        $ModuleVersion
+        $ModuleVersion,
+        
+        [Switch]
+        $AllVersions
     )
 
-    $modules = Get-Module -ListAvailable
+    #Only get the latest version of each module
+    $modules = Get-Module -ListAvailable 
+    if (-not $AllVersions) {
```

Comment:
Minor nit, the code style for this project is to use `!`.  Could you change this to `!$AllVersions`?

Reply:
Done.